R/pkg/R/generics.R

```diff
@@ -799,6 +799,10 @@ setGeneric("write.df", function(df, path = NULL, ...) { standardGeneric("write.d
 #' @export
 setGeneric("randomSplit", function(x, weights, seed) { standardGeneric("randomSplit") })
 
+#' @rdname broadcast
+#' @export
+setGeneric("broadcast", function(x) { standardGeneric("broadcast") })
```

Comment:
let's fix up the sorting when 2.2.0 is released - it would help to minimize major changes for now to make it easier to merge fixes.